### ISSUE TITLE
TypeError: bcrypt.hashSync is not a function

### ISSUE THREAD

**Issue by rzulian** [JamesBroadberry/deno-bcrypt#12]
I'm getting this error and I cannot understand what's wrong.
I've tried also to work with the async version, but I'm getting the same error.

```TypeError: bcrypt.hashSync is not a function
    at Object.hashIt (hash.ts:4:19)
    at store (UserController.ts:29:26)
    at async dispatch (middleware.ts:41:7)
    at async dispatch (middleware.ts:41:7)
    at async dispatch (middleware.ts:41:7)
    at async Application.#handleRequest (application.ts:252:9)``

**Comment by JamesBroadberry** [comment]
Hi,

Could you provide a example your code that causes this issue so I can take a look please?

Unfortunately those logs don't provide me with enough information to be able to help.

Thanks,
James

**Comment by rzulian** [comment]
Hi,
I've pushed this repository https://github.com/rzulian/deno-mongo

You can replicate the problem with a post `localhost:8000/user`
`body:
{
    "username": "name",
    "email": "bla@abc.coml",
    "password": "test"
}

**Comment by JamesBroadberry** [comment]
Perfect thanks, could you clarify what you're using to run the application too (including flags) please? 
I'll have a look a little later on.

**Comment by rzulian** [comment]
I'm using these flags `deno run --allow-net --allow-write --allow-read --allow-env --allow-plugin --unstable main.ts`

**Comment by JamesBroadberry** [comment]
I just cloned your repo, removed all the DB actions so I don't have to run mongo and left the "POST" method. I added a console.log after `const pwd = await bcrypt.hash(value.password);` to log out `pwd` and it does it succesfully.

I'm not sure what the issue is that you're having but I don't think it's something to do with this bcrypt module.

Have you updated deno to the latest version (`deno upgrade`) and reloading your cached modules (`deno cache reload main.ts`)?

Hopefully that should sort you out but if it doesn't you might find some better help in the [Deno discord](https://discord.gg/deno)
